---
a1: 1
a2: 1
d3: 0.5
alpha2: 1.57
theta1: 0
theta2: 0
joint1_lim: 0.5
joint2_lim: 0.5
joint3_lim: 0.5
joint1: {
 rpy: "0 -0 0",
 pos: "0 -0 0",
}
joint2: {
 rpy: "0 -0 0",
 pos: "1 -0 0",
}
joint3: {
 rpy: "1.57 -0 0",
 pos: "1 -0.5 0.000398163",
}
...